src/istio/ambient/zarf.yaml

```
@@ -0,0 +1,74 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+kind: ZarfPackageConfig
+metadata:
+  name: uds-core-istio-ambient
+  description: "UDS Core Istio Ambient"
+  url: https://istio.io/latest/
+variables:
+  - name: CNI_CONF_DIR
+    description: "CNI configuration directory"
+    default: ""
+  - name: CNI_BIN_DIR
+    description: "CNI binary directory"
+    default: ""
+
+components:
+  - name: istio-ambient
+    required: true
```

I think it still works as expected but for consistency sake and peace of mind, maybe this should be false?
```suggestion
    required: false
```